tailwind.config.js

```diff
@@ -1,7 +1,13 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-	content: ["./src/**/*.{ts,tsx}"],
+	content: ["./src/**/*.{ts,tsx,js,jsx}"],
 	theme: {
+		colors: {
+			"red": "#f87171",
+			"yellow": "#fcd34d",
+			"green": "#86efac",
+			"button-text-color": "#f9fafb",
```

Comment:
The name `button-text-color` is somewhat specific and might be limiting if you later want to use this color in other contexts beyond buttons. This is because it would appear on usage as `text-button-text-color`

Maybe `primary-light` or `base` would be better.

Reply:
Fixed